src/Payments/SepaBankTransfer.res

```diff
@@ -60,6 +60,16 @@ let make = (
   }, (email, fullName, country))
   useSubmitPaymentData(submitCallback)
 
+  let updatedOptionsArrayForCountry: array<
+    DropdownField.optionType,
+  > = countryNames->Array.map(item => {
+    let optionVal: DropdownField.optionType = {
```

Comment:
Please call the common function here as well

Reply:
Updated